graph/tags.go

```diff
@@ -231,10 +240,12 @@ func (store *TagStore) Delete(repoName, ref string) (bool, error) {
 	return deleted, store.save()
 }
 
+// Tag a image.
 func (store *TagStore) Tag(repoName, tag, imageName string, force bool) error {
 	return store.SetLoad(repoName, tag, imageName, force, nil)
 }
 
+// SetLoad store the image to the store.
```

Comment:
s/store/stores/
Probably should add that it's checking few things (like if that repoName:tag already exists and such).